pkg/scm/git/url.go

```diff
@@ -106,6 +108,14 @@ func Parse(rawurl string) (*URL, error) {
 		}, nil
 	}
 
+	// if ssh://git@github.com:sclorg/nodejs-ex then strip ssh:// a la what we
+	// see in other upstream git parsing handling of scp styled git URLs;
+	//TODO is there someway to infer git v1 vs. v2 so we do not have to strip this
+	// for older clients
```

Comment:
@adambkaplan let's discuss this in scrum if not in line here

Reply:
Per discussion on slack - we shouldn't worry about git version.